go.mod

```diff
@@ -43,7 +43,7 @@ replace (
 	github.com/aws/eks-anywhere/release => ./release
 	github.com/containerd/containerd => github.com/containerd/containerd v1.5.9
 	github.com/dgrijalva/jwt-go => github.com/golang-jwt/jwt/v4 v4.0.0
-	github.com/docker/distribution => github.com/docker/distribution v2.8.0-beta.1+incompatible
+	github.com/docker/distribution => github.com/docker/distribution v0.0.0-20220207154021-dcf66392d606
```

Comment:
Will Dependabot know this is v2.8.0+?

Reply:
I think it did give us an option to use a commit or a release tag. So it might have a way to figure it out?

Reply:
Possibly not: dependabot/dependabot-core#3017 😢

Reply:
Or maybe this is just for automatic updates, not sure.